nss-day-cohort-29/nutshell-spirit-wind
# Db friendsetup

# Description

Friends are being targeted correctly based on currentUserId. They are not printing to DOM correctly yet but you can see the friendsList in the console log. I still have to filter the users to get userName and email to appear on DOM. Different users will have different quantities of friends in the console.log


## Type of change

Please delete options that are not relevant.




# Testing Instructions for Change Made

git fetch --all
git checkout db-friendsetup
cd src/lib
grunt
make sure to populate database.json with friends and users with appropriate data in each resource
login/register user


# Checklist:
